beacons-data/export_legacy_beacons.sql

```diff
@@ -0,0 +1,19 @@
+SELECT 
+	hex_id, 
+	UPPER(data -> 'beacon' ->> 'departRefId')as dept_ref, 
+	UPPER(beacon_status),
+	TO_CHAR(created_date :: DATE, 'dd/mm/yyyy') as created_date, 
```

Comment:
This needs to be first registration date, not created date. 

Created date on legacy only goes back as far as 2015 when a migration happened.